### ISSUE TITLE
How to change batch normalization momentum during training

### ISSUE THREAD

**Issue by alex-epp** [keras-team/keras#12230]
I'm trying to duplicate the results from https://github.com/charlesq34/pointnet, which involves changing the batch normalization decay rate during training. Is there a way to do this in keras?
